### ISSUE TITLE
Duplicate translated strings

### ISSUE THREAD

**Issue by daniele-athome** [nijel/weblate#889]
I'm using Weblate 2.4 and got into a duplicate strings issue. I'm using Android.

```
<string name="msg_auth_error">L\\\'autenticazione ad un server di Kontalk è fallita durante l\\\'ultimo tentativo di connessione. Potrebbe essere un problema temporeaneo del server oppure, più probabilmente, il server sta rifiutando la tua chiave personale. Ti raccomandiamo di cancellare il tuo account e di registrarti nuovamente (tuttavia perderai tutti i tuoi messaggi).<b>ATTENZIONE!</b>\nL\'autenticazione ad un server di Kontalk è fallita durante l\'ultimo tentativo di connessione. Potrebbe essere un problema temporeaneo del server oppure, più probabilmente, il server sta rifiutando la tua chiave personale. Ti raccomandiamo di cancellare il tuo account e di registrarti nuovamente (tuttavia perderai tutti i tuoi messaggi).</string>
```

Now, I won't get into the triple-escaped backslash (that might be a translate-toolkit bug, I'm still fighting with it despite using the latest version, but that's another issue), but starting from `<b>`, the rest is an old translation. Somehow, they got joined together. Modifications to the resource files are done exclusively with Weblate (so no git conflict is possible), and Weblate itself is configured to use rebase.

Fixing the string in Weblate doesn't work and no changes are detected in the git working copy. Even if I alter the translation, the "old part" is still in git but I can't see it in Weblate. For example:

In Weblate I see this translation:
```
L\\\'autenticazione ad un server di Kontalk è fallita durante l\\\'ultimo tentativo di connessione. Potrebbe essere un problema temporeaneo del server oppure, più probabilmente, il server sta rifiutando la tua chiave personale. Ti raccomandiamo di cancellare il tuo account e di registrarti nuovamente (tuttavia perderai tutti i tuoi messaggi).
```

But in git, strings.xml has:
```
<string name="msg_auth_error">L\\\'autenticazione ad un server di Kontalk è fallita durante l\\\'ultimo tentativo di connessione. Potrebbe essere un problema temporeaneo del server oppure, più probabilmente, il server sta rifiutando la tua chiave personale. Ti raccomandiamo di cancellare il tuo account e di registrarti nuovamente (tuttavia perderai tutti i tuoi messaggi).<b>ATTENZIONE!</b>\nL\'autenticazione ad un server di Kontalk è fallita durante l\'ultimo tentativo di connessione. Potrebbe essere un problema temporeaneo del server oppure, più probabilmente, il server sta rifiutando la tua chiave personale. Ti raccomandiamo di cancellare il tuo account e di registrarti nuovamente (tuttavia perderai tutti i tuoi messaggi).</string>
```

Since the Weblate strings are the correct ones, is there a way to massively update my resource files with the translations from the Weblate database?

**Comment by nijel** [comment]
This still sounds like some bug in translate-toolkit, but it needs to be tested deeper.

**Comment by nijel** [comment]
I think this should be fixed as of 82840a35.
